client/decrypt/decrypt.go

```diff
@@ -0,0 +1,102 @@
+package decrypt
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/tendermint/tendermint/libs/cli"
+	"io/ioutil"
+)
+
+const (
+	FlagCipherText     = "text"
+	FlagCipherTextPath = "path"
+)
+
+func Command(defaultNodeHome string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "decrypt [name] (--text [ciphertext] | --path [ciphertext-file])",
+		Short: "Decrypt and output the ciphertext file encrypted with ECDSA PublicKey.",
+		Long: `This command can decrypt ciphertext encrypted with ECDSA PublicKey. 
+The file contents must be Standard Base64 encoded.
```

Comment:
I think it would be better to not force users to prepare base64-encoded files. In my opinion, the file needs to contain a plain byte stream that is encrypted, so that users who have files that are not base64-encoded can use this tool.
If the user has a base64-encoded encrypted file, he/she can this tool as below:
```bash
cat my-b64-enc-file.txt | base64 -d > my-enc-file.txt
panacead decrypt my-key --path my-enc-file.txt
```

Reply:
OK. I understand.
I will fix it soon.

Reply:
I fixed it.
https://github.com/medibloc/panacea-core/pull/355/commits/7b214daa7fbdfdd02198eb42c689c280d1128154